index.html

```diff
@@ -19,169 +19,296 @@
+                <li class="card-detail canopy">CANOPY</li>
+                <li class="card-detail option">
+                  <img src="assets/bullets.png" alt="bullet" class="circle" />
+                  Back End Dev
+                </li>
+                <li class="card-detail option">
+                  <img src="assets/bullets.png" alt="bullet" class="circle" />
+                  2015
+                </li>
+                <li class="card-detail dcanopy2_facebook uber">Uber</li>
+                <li class="card-detail optiond2_full">
+                  <img src="assets/bullets.png" alt="bullet" class="circle" />
+                  Lead Developer
+                </li>
+                <li class="card-detail optiond2_full">
+                  <img src="assets/bullets.png" alt="bullet" class="circle" />
+                  2018
+                </li>
+              </ul>
+              <p class="instruction">
+                A daily selection of privtely personalized reads; no accounts or
+                sign-ups required.
+              </p>
+              <p class="instruction-show">
+                A smart assistant to make driving mode safe, efficient, and fun
+                by unlocking your most expensive computer: your car.
+              </p>
+              <div class="language1 inline">
+                <ul>
+                  <li class="langu">html</li>
+                  <li class="langu">css</li>
+                  <li class="langu">javascript</li>
+                  <li class="langu2">html</li>
+                  <li class="langu2">Ruby on rails</li>
+                  <li class="langu2">css</li>
+                  <li class="langu2">javascript</li>
+                </ul>
+              </div>
+            </div>
+            <button type="button" class="btn-one">See project</button>
+          </div>
         </div>
       </section>
-      <section class="about-me-card">
-        <div class="container">
-          <h1 class="intro-text">About Myself!</h1>
-          <p class="intro-text-2">
-            Hello I’m a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
-            hestiate to contact me.
-          </p>
-          <div class="connect">
-            <p class="icons-text common-color">LET’S CONNECT</p>
-            <ul class="social-icons">
+      <section class="section-about" id="about-myself">
+        <div class="about-grid">
+          <div class="headline-text">
+            <h1>About Myself</h1>
+
+            <p class="paragraph1 para-about">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
```

Comment:
- [ ] I think it could be a good idea to apply % to this sections because the left side is taking more space than the right side of this section, as opposed to the Figma design where the left side takes around 40% and the right side takes 60% :wink:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144635044-310c43ec-fdc5-45e7-b5bd-df769f25b068.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144636620-0d770f64-a111-4db1-b024-ceb2d31c1d01.png)

- [ ] I would also suggest adding more space to the sides of the section because yours is taking almost the entire width. It will make your portfolio look even better :smiley: